### DISCUSSION TITLE
챌린지 기념품 안내 메일, 주소 입력에 대해 문의드립니다.

### DISCUSSION THREAD

**Discussion by solidcellaMoon** [devrel-kr/HackaLearn#428] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요! 멋진 행사 진행해주셔서 감사합니다. 정말 즐겁게 진행했습니다.
챌린지 기념품 및 수상 안내 메일을 받고, 몇가지 문의 드릴 사항이 생겨 이렇게 글을 올립니다.

1. 기념품 주소 기입란에 대하여
주소 입력폼에서 우편번호를 기입할 때, 첫자리가 0으로 시작하는 경우
입력 후 0이 사라지는 현상을 발견했습니다.
(예시: 우편번호가 03142인 경우, 텍스트창에 기입하면 3142로 입력됨)
이러한 현상은 무시하고 주소를 제출해도 괜찮을까요?

2. 클라우드 스킬 챌린지 수상 대상자인데, 기념품 안내 메일을 받지 못한 경우
저희 팀의 팀원 2분께서 챌린지 수상 명단에는 등록되어있는데,
수상 안내 메일은 받았지만 기념품 안내 관련 메일을 받지 못한 상태입니다.
스팸메일함도 확인하셨다고 합니다.
@vilut1002 님과 @minie12 님께 메일이 전송되었는지 확인해주셨으면 합니다.

마지막으로, 이번 해커런 행사를 진행해주셔서 정말 감사드립니다.
다음 기회에도 다시 참가할 수 있다면 좋겠습니다!

**Comment by justinyoo** [comment]
@vilut1002 클라우드 스킬 챌린지 관련 이메일이 새벽 1시 4분 경 발송되었습니다. 확인해 보세요

![image](https://user-images.githubusercontent.com/1538528/130209105-3d49b6c2-b03d-4b6d-9208-84b041314360.png)

**Comment by justinyoo** [comment]
@minie12 마찬가지로 새벽 1시 4분 경에 클라우드 스킬 챌린지 관련 이메일이 발송되었습니다. 확인해 보세요.

![image](https://user-images.githubusercontent.com/1538528/130209430-c7c728c6-a3dd-415d-bfef-de4590ee10e5.png)

**Comment by justinyoo** [comment, marked as answer]
@solidcellaMoon 우편번호 맨 앞자리가 0일 경우 사라지는 건 저희쪽에서 알아서 처리하니 걱정마시고 입력하세욥!

**Reply by justinyoo** [reply]
다시 한 번 @solidcellaMoon @vilut1002 @minie12 모두 수상을 축하드리고, 다음에 다시 이런 행사가 열린다면 또 만나요!

**Reply by jung-youjin** [reply]
@solidcellaMoon @vilut1002 @minie12 수상 너무 축하드립니다!!!🥳 다음에 또 뵐 수 있길 바라며..!!😚🙏🏼

**Reply by solidcellaMoon** [reply]
안녕하세요! 두분 모두 메일 잘 확인하셨다고 합니다. 운영진분들 답변과 축하 모두 감사드립니다!!!😄
